test-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2021 Canonical Ltd.
```

Comment:
You could inline the contents of this file directly in your tox config?

Reply:
I agree, I just did it this way for the sake of consistency across all our repositories.